deflate.h

```diff
@@ -194,7 +194,8 @@ typedef struct internal_state {
 
     int nice_match; /* Stop searching when current match exceeds this */
 
-#ifdef X86_PCLMULQDQ_CRC
+#if defined(__x86_64__) || defined(_M_X64) || defined(__i386) || defined(_M_IX86)
```

Comment:
Why not just remove this #define altogether?

Reply:
Well, if those will never be used by any other arch, then they are just bloat.
If it is likely that we want to use them for other arches as well, then sure, we could keep them around.

Reply:
Ok